utils.sh

```diff
@@ -22,6 +22,54 @@ done
 sudo cp "$current" "$target"
 }
 
+function patch_node_ignition() {
+    local kind
+    local bootstrap
+    local wd
+    local config
+
+    kind="$1"
+    bootstrap="$2"
+    wd="$(mktemp -d)"
+    ssh_opts=("-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null")
+
+    # Wait for the release config to be pulled
+    while
+        config=$(ssh "${ssh_opts[@]}" "core@$bootstrap" \
```

Comment:
here and in other places where we `ssh` to the coming up host - would it make sense to tackle ssh `ConnectionAttempts` and/or `ConnectTimeout` options  in hope for more predicable timing?